### PR TITLE
Add eslint, prettier

### PR DESCRIPTION
**Pull request by danielnixon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+install:
+  - yarn install --frozen-lockfile
+script:
```
Comment (robinMcA) [inline .travis.yml:4]: Why not git hub actions?

File: package.json [hunk 2]
```diff
@@ -19,11 +19,18 @@
...
+    "jasmine": "^3.5.0",
+    "prettier": "^1.19.1",
     "serverless": "^1.53.0",
     "serverless-offline": "^5.11.0"
   },
   "scripts": {
-    "sls": "serverless"
+    "sls": "serverless",
+    "lint": "eslint src",
+    "build": "sls package"
```
Comment (robinMcA) [inline package.json]: Probably better to use the full string `serverless`?

**Review by robinMcA — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (danielnixon) [reply .travis.yml:4]: Travis was what I had within copy+paste reaching distance

File: package.json (hunk 2 above)
Comment (danielnixon) [reply package.json]: Done

**Review by PouriaSterling — APPROVED** [review]

**Pull request merged**